benchcab/utils/pbs.py

```diff
@@ -3,46 +3,35 @@
 
 """Contains helper functions for manipulating PBS job scripts."""
 
-from typing import Optional
-
 from benchcab import internal
+from benchcab.config import PBSConfig
 
 
 def render_job_script(
     project: str,
     config_path: str,
     modules: list,
     benchcab_path: str,
+    pbs_config: PBSConfig,
     verbose=False,
     skip_bitwise_cmp=False,
-    pbs_config: Optional[dict] = None,
 ) -> str:
     """Returns the text for a PBS job script that executes all computationally expensive commands.
 
     This includes things such as running CABLE and running bitwise comparison jobs
     between model output files.
     """
-    if pbs_config is None:
-        pbs_config = internal.FLUXSITE_DEFAULT_PBS
-
     module_load_lines = "\n".join(
         f"module load {module_name}" for module_name in modules
     )
     verbose_flag = "-v" if verbose else ""
-    ncpus = pbs_config.get("ncpus", internal.FLUXSITE_DEFAULT_PBS["ncpus"])
-    mem = pbs_config.get("mem", internal.FLUXSITE_DEFAULT_PBS["mem"])
-    walltime = pbs_config.get("walltime", internal.FLUXSITE_DEFAULT_PBS["walltime"])
-    storage_flags = [
-        "gdata/ks32",
-        "gdata/hh5",
-        "gdata/wd9", # Subgroup of gdata/ks32
-        *pbs_config.get("storage", internal.FLUXSITE_DEFAULT_PBS["storage"]),
-    ]
+    # wd9 is subgroup of gdata/ks32
```

Comment:
no. Why this comment??

Reply:
I thought the documentation mentioned only 2 projects at NCI, and a new developer would be confused with `wd9` (but can remove this comment for now)